tests/unit/amo/reducers/test_guides.js

```diff
@@ -28,14 +33,37 @@ describe(__filename, () => {
       expect(newState.loading).toEqual(false);
     });
 
-    it('stores the add-on GUIDs', () => {
+    it('stores the add-on GUIDs by slug', () => {
+      const slug = 'some-slug';
       const guids = ['test', 'test2'];
       const state = guidesReducer(
         undefined,
-        fetchGuidesAddons({ guids, errorHandlerId: 'test' }),
+        fetchGuidesAddons({ slug, guids, errorHandlerId: 'test' }),
       );
 
-      expect(state.guids).toEqual(guids);
+      expect(state.bySlug).toEqual({
```

Comment:
We tend to use the selectors in the tests, so this might be better using `getGUIDsBySlug` in the assertion rather than inspecting the internal state.